example/lib/widgets/buy_sheet.dart

```diff
@@ -152,7 +151,8 @@ class BuySheetState extends State<BuySheet> {
         context: BuySheet.scaffoldKey.currentContext!,
         title: title,
         description:
-            "Check your console for a CURL command to charge the nonce, or replace CHARGE_SERVER_HOST with your server host.");
+            "Check your console for a CURL command to charge the nonce, or replace CHARGE_SERVER_HOST with your server host.",
+        status: false);
```

Comment:
We should celebrate the developer made this far. Showing a failure icon isn't encouraging. Can we make it a success message?